tests/interactive.fish

```diff
@@ -39,7 +41,7 @@ if not type -q expect
     exit 0
```

Comment:
There's a block here which exits with success if `expect` isn't installed; possibly the same should be done with `python3 -c 'import pexpect'`.

Reply:
done